.gitattributes

```diff
@@ -0,0 +1 @@
+/src/**/* linguist-generated=true
```

Comment:
oh nice, TIL

Reply:
@jonatanklosko what do you think about marking the `bindings/` dir as generated as well? looks like linguist currently thinks tree-sitter-iex is a rust project :smile:

Reply:
Haha, nice one, will do!